agilent_infiniium_scope.py

```diff
@@ -31,39 +31,39 @@
 """
 
 
-
 from labrad import types as T, util
 from labrad.server import setting
 from labrad.gpib import GPIBManagedServer, GPIBDeviceWrapper
 from twisted.internet.defer import inlineCallbacks, returnValue
 import labrad.units as U
 from struct import unpack, calcsize
-
-import numpy, re
+import numpy as np
 
 COUPLINGS = ['AC', 'DC', 'GND']
-TRIG_CHANNELS = ['AUX','CH1','CH2','CH3','CH4','LINE']
+TRIG_CHANNELS = ['AUX', 'CH1', 'CH2', 'CH3', 'CH4', 'LINE']
 VERT_DIVISIONS = 8.0
 HORZ_DIVISIONS = 10.0
 SCALES = []
 
+
 class AgilentDSO91304AServer(GPIBManagedServer):
     name = 'Agilent Infiniium Oscilloscope'
-    deviceName = ['Agilent Technologies DSO91304A', 'KEYSIGHT TECHNOLOGIES DSO90804A']
-        
+    deviceName = ['Agilent Technologies DSO91304A',
+                  'KEYSIGHT TECHNOLOGIES DSO90804A']
+
     @setting(11, returns=[])
     def reset(self, c):
         dev = self.selectedDevice(c)
-        yield dev.write('*RST')
+        yield dev.query('*RST;*OPC?')
         # TODO wait for reset to complete
```

Comment:
Not technically... *OPC?  just waits for the scope to parse the commands... Keysight's applicaiton engineer said that it is "usually good enough" and didn't suggest any better method.

There may or may not be some status bit we could check, but I don't think I was having an issue before adding the *OPC? query and adding it should just make things better.